.github/workflows/workflow.yaml

```diff
@@ -105,7 +105,7 @@ jobs:
 
       - name: Pytest
         run: |
-          micromamba run -n modyn pytest modyn --cov-reset --cache-clear --cov-fail-under=90
+          micromamba run -n modyn pytest modyn --cache-clear --cov-report term --cov-fail-under=90
```

Comment:
why do we need --cov-reset? This is the first time we use pytest and no previous configurations are accumulated

Reply:
This stems from times when we were using internal Github runners (non public) where the cache was not properly cleaned. I would suggest to just keep it, if it's not causing trouble.

Reply:
I need to investigate into `--cov-reset`. Theoretically it has nothing to do with cache 
https://github.com/pytest-dev/pytest-cov/issues/460